manim/mobject/svg/brace.py

```diff
@@ -104,10 +114,7 @@ def get_tex(self, *tex, **kwargs):
         return tex_mob
 
     def get_tip(self):
-        # Very specific to the LaTeX representation
-        # of a brace, but it's the only way I can think
-        # of to get the tip regardless of orientation.
-        return self.get_all_points()[self.tip_point_index]
+        return self.points[7 * 4]
```

Comment:
```suggestion
        return self.points[28]
```
Also, I don't like that this is hardcoded, but it'd be good to write a comment on what this is doing, or a docstring.